application/helpers/expressions/em_manager_helper.php

```diff
@@ -8548,8 +8550,17 @@ static function ProcessCurrentResponses()
                                     }
                                 }
                                 break;
-#                            case 'N': //NUMERICAL QUESTION TYPE
-#                            case 'K': //MULTIPLE NUMERICAL QUESTION
+                            case 'N': //NUMERICAL QUESTION TYPE
+                            case 'K': //MULTIPLE NUMERICAL QUESTION
+                                if (trim($value)!="") {
+                                    $aAttributes=$LEM->getQuestionAttributesForEM($LEM->sid, $qid, $_SESSION['LEMlang']);
+                                    if (!isset($aAttributes[$qid])) {
+                                        $aAttributes[$qid]=array();
+                                    }
+                                    if($aAttributes[$qid]['thousands_separator']=='1') {
+                                        $value=str_replace($invertRadix['separator'], '', $value);
+                                    }
+                                }
```

Comment:
This part permit to remove the dots or commats from the value. (2,300,400.10 => 2300300.10 or 2.300.400,10 => 2300300,10).
Without this part, I had SQL crashes :

    With dot as decimal mark :
    UPDATE lime_survey_499425 SET "lastpage"=0, "499425X8X40"='23,123,131,231.23', "499425X8X41"='1,231,232,131,231' WHERE ID=6
    465,465,465,443 is not a decimal value

Or

    With commat as decimal mark :
    UPDATE lime_survey_499425 SET "lastpage"=0, "499425X8X40"='1.231.231.23', "499425X8X41"='131.231.231.231.232' WHERE ID=5
    This is the same with '131.231.231.231.232'

But you are right, with decimal mark as a dot and a numerical input with thousands_separator = 1 and interger_only = 0, the value saved is wrong : 100022 instead of 1000.22